charts/stable/qbittorrent/values.yaml

```diff
@@ -36,30 +36,11 @@ service:
         port: 8000
         targetPort: 8000
         protocol: http
-workload:
-  qbitportforward:
-    enabled: true
-    type: Deployment
```

Comment:
Just change this type here,
Add schedule and set it to. {{ .Values.some.new.field.for.schedule }}

Remove rolling update
Remove replica

Add command

Profit


(No need for new files)

Reply:
type is cronjob?

Reply:
CronJob yes